### PR TITLE
12278 show claim polling

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
added polling behavior to claims detail 

## Testing done
local testing against beta mocks user- navigate to `http://localhost:3001/track-claims/` after logging in. May need to set `mock_claims: true` in `vets-api` `settings.yml` and to set a default betamocks response for `evss_claims_async` following [these](https://dsva.slack.com/archives/C446YB1K8/p1534433342000100) instructions: 

> go to local vets-api-mockdata repo to `evss/claims/index` and `show`, duplicate the file that is there and rename the duplicated file as `default.yml`

## Testing Plan
none

## Screenshots
No familiar with existing behavior but this is what I see locally. screen shot 2018-08-20 at 16 31 02

## Acceptance Criteria (Definition of Done)
Behavior should be identical to what's in production for the claims details page. 

#### Unique to this PR
`¯\_(ツ)_/¯`

#### Applies to all PRs

- [x] Appropriate logging
- [x] Documentation has been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: src/applications/claims-status/tests/actions.unit.spec.js [hunk 1]
```diff
@@ -415,73 +490,6 @@ describe('Actions', () => {
...
-          done();
-        }
-      };
-
-      thunk(dispatch);
-    });
-    it('should fail on 500 error', (done) => {
-      const claim = { data: {}, meta: {} };
-      fetchMock.returns({
-        'catch': () => ({ then: (fn) => fn({ ok: false, status: 500, json: () => Promise.resolve(claim) }) }),
```
Comment (rianfowler) [inline src/applications/claims-status/tests/actions.unit.spec.js]: 🔥 🐴

File: src/applications/claims-status/actions/index.jsx [hunk 2]
```diff
@@ -171,7 +172,7 @@ export function pollClaimsStatus({
       }
 
       setTimeout(
-        pollClaimsStatus,
+        pollRequest,
         pollingInterval,
         { onError, onSuccess, pollingInterval, request, shouldFail, shouldSucceed }
```
Comment (erikphansen) [inline src/applications/claims-status/actions/index.jsx]: Does `target` not need to be passed to the delayed call to `pollRequest`?

**Review by erikphansen — APPROVED** [review]
I had one small question.

File: src/applications/claims-status/actions/index.jsx (hunk 2 above)
Comment (rianfowler) [reply src/applications/claims-status/actions/index.jsx]: whoops

**Pull request merged**